src/TLuaInterpreter.cpp

```diff
@@ -10922,8 +10922,8 @@ int TLuaInterpreter::getSpecialExits(lua_State* L)
             } else {
                 exit = dir;
             }
-            lua_pushstring(L, exit.toLatin1().data());       //done to remove the prepended special exit status
-            lua_pushstring(L, exitStatus.toLatin1().data()); //done to remove the prepended special exit status
+            lua_pushstring(L, exit.toUtf8().constData());       //done to remove the prepended special exit status
+            lua_pushstring(L, exitStatus.toUtf8().constData()); //done to remove the prepended special exit status
```

Comment:
It might be worth checking - I think the `exitStatus` part (which is actually `"1"` or `"0"` depending on whether the special exit is locked or not) is redundant but it might be masking the actually wanted bit that is the `exit` name...!